src/api_server/swagger/firecracker.yaml

```diff
@@ -464,6 +464,13 @@ paths:
       description:
         Enables an entropy device that provides high-quality random data to the guest.
       operationId: putEntropyDevice
+      parameters:
+        - name: body
+          in: body
+          description: Guest entropy device properties
+          required: false
```

Comment:
I don't really know my way around the API server, but is the body of this PUT request really optional? This code 

```rs
pub(crate) fn parse_put_entropy(body: &Body) -> Result<ParsedRequest, Error> {
    let cfg = serde_json::from_slice::<EntropyDeviceConfig>(body.raw())?;
    Ok(ParsedRequest::new_sync(VmmAction::SetEntropyDevice(cfg)))
}
```

seems to imply that its not (e.g. I can't just put `null` in the body)

Reply:
I think that what does the trick here is that `EntropyDeviceConfig` includes a `Option<RateLimiterConfig>`.

Reply:
Mh, I'm not sure, there's a difference between having `body = {rate_limiter: null}` and `body = null`

Reply:
hmmm, you are right. Taking a look.

Reply:
Ok fixed!